libs/zzz/solver/src/calc.ts

```diff
@@ -92,7 +101,9 @@ export function applyCalc(
   sum['final_atk'] = s('initial_atk') * (1 + s('cond_atk_')) + s('cond_atk')
   sum['final_def'] = s('initial_def') * (1 + s('cond_def_')) + s('cond_def')
   sum['impact'] = s('impact') * (1 + s('impact_'))
-  sum['anomMas'] = s('anomMas') * (1 + s('anomMas_'))
+  sum['anomMas'] =
```

Comment:
there is also base anomMas from core skill (like yanagi). technically not super important since there is no optimization impact on damage from anomMas, but the displayed stat value will be wrong